packages/forms/frontend/sirius-components-widget-reference/src/components/ModelBrowserTreeView.tsx

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { TreeView } from '@eclipse-sirius/sirius-components-trees';
+import { makeStyles } from '@material-ui/core/styles';
+import { useState } from 'react';
+import { ModelBrowserFilterBar } from './ModelBrowserFilterBar';
+import { ModelBrowserTreeViewProps } from './ModelBrowserTreeView.types';
+
+const useTreeStyle = makeStyles((_) => ({
+  ul: {
+    width: 'inherit',
+    minWidth: 'fit-content',
+  },
+  title: {
+    opacity: 0.6,
+    fontSize: 12,
+  },
+  borderStyle: {
+    border: '1px solid #0000008A',
+    height: 300,
+    overflow: 'auto',
+  },
+}));
+
+export const ModelBrowserTreeView = ({
+  editingContextId,
+  selection,
+  setSelection,
+  widget,
+  markedItemIds,
+}: ModelBrowserTreeViewProps) => {
+  const classes = useTreeStyle();
+
+  const [filterBarText, setFilterBarText] = useState<string>('');
```

Comment:
Since I'm 99% sure that your view will become more complex over time, create a state with a proper type like we always do

Reply:
done